src/types.ts

```diff
@@ -10,26 +10,32 @@ export type SquareClasses =
 export type SquareSize = "small" | "big";
 
 export type SelectedSquare = {
+    id: string;
+    image: string;
+    setClassName?: SetStateFunction<SquareClasses>;
+};
+
+export type NullableSelectedSquare = {
```

Comment:
Maybe we should just create a generic `Nullable` type that can be reused like:
`export type Nullable<T> = T | null;` 
That way we can use it as: `Nullable<SelectedSquare>`

Reply:
done